pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "trtools"
+authors = [
+    {name = "Melissa Gymrek", email = "mgymrek@ucsd.edu"},
+]
```

Comment:
If this is going to be here, we should add Nima Mousavi, myself, you and others(?) (Aarushi if her tool has been published) to the list.

Reply:
ok, done!